backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/BpkAppSearchModal.kt

```diff
@@ -95,13 +96,14 @@ fun BpkAppSearchModal(
         title = title,
         onDismiss = onClose,
     ) {
-        BpkAppSearchModalImpl(
-            inputText = inputText,
-            inputHint = inputHint,
-            results = results,
-            onInputChanged = onInputChanged,
-            clearAction = clearAction,
-            behaviouralEventWrapper = behaviouralEventWrapper,
-        )
+        Column {
```

Comment:
why this change?

Reply:
A remnant from a previous way of implementing the change. Will remove